config/env/development.js

```diff
@@ -65,5 +65,6 @@ module.exports = {
       }
     }
   },
-  livereload: true
+  livereload: true,
+  seedDB: true
```

Comment:
It would be nice to allow this to be overriden from the console through an environment variable for quick switching